src/pages/en/community/nsfw-policy.mdx

```diff
@@ -0,0 +1,206 @@
+
+### Suggestive or sexual attention
+
+- Suggestive comments about (or reactions to) other users that haven't consented to it, especially if they've stated they're uncomfortable with it or aren't old enough -- regardless of how explicit the comments are
+- Sexual comments about other users, regardless of context or whether they consent to it, including implications about other people or groups of people
+- Any suggestive or intimate depictions involving people other than the poster, without their consent or knowledge -- including creep-shots or images reasonably assumed to be private, regardless of how well-known the person is
+
+### Suggestive or sexual content
+
+- Imagery or other media (including audio) intended to be suggestive or arousing, or to seek a sexual response from another user
+- Explicit sexual content and behaviour, including role-play and innuendo intended to imply sexual intent
+- Depictions of movements intended to be sexual or seductive, including twerking (even if it's [a Crewmate from Among Us](https://among-us.fandom.com/wiki/Crewmate)), regardless of the reason it's being posted
+
+### Other content
+
+- Depictions of anything that can reasonably be considered NSFL - animal abuse, gore, serious injuries, body horror, suicide, and so on
+
+</Foldable>
+
+## Moderation Policy
+
+First and foremost, we'd like to point out that we're not trying to be the fun police. Our community is host to a diverse range of people with many backgrounds, and some of them may suffer from panic attacks or other strong reactions to NSFW content. We believe that these people have a place in our community, and we'd like to make it as welcoming for them as we can.
+
+We're aware of the different levels of tolerance people have for drawn art versus photography and depictions of real people, and we understand the need to take content _and_ context into account when moderating this type of media. We'll do our best to avoid under-moderating while still allowing an acceptable degree of freedom.
+
+Additionally, it's worth pointing out that NSFW content can present a safety issue when posted around minors. Our team is intimately familiar with the correlation between people posting NSFW content around minors and how those people can be a danger to those minors, and we will not tolerate any behaviour that impacts the safety of our users.
+
+While we moderate NSFW content on a case-by-case basis in accordance with our broader moderation policy, we'd like to point out that the Community Team may include minors and people that have strong reactions to NSFW content. For this reason, **we do not allow minors to handle NSFW content,** and Community Team members that do not wish to handle NSFW content **are welcome to avoid doing so**, or ask another staff member to handle it in their stead.
```

Comment:
```suggestion
While we moderate NSFW content on a case-by-case basis in accordance with our broader moderation policy, we'd like to point out that the Community Team may include minors and people that have strong reactions to NSFW content. For this reason, **we do not allow minors to handle NSFW content,** and Community Team members that do not wish to handle NSFW content **are welcome to avoid doing so**, or ask another staff member to handle it instead.
```